pkg/triggers/watcher.go

```diff
@@ -0,0 +1,186 @@
+package triggers
+
+import (
+	"context"
+
+	"github.com/google/go-cmp/cmp"
+	testtriggers_v1 "github.com/kubeshop/testkube-operator/apis/testtriggers/v1"
+	"github.com/kubeshop/testkube-operator/pkg/informers/externalversions"
+	"github.com/kubeshop/testkube-operator/pkg/validation/tests/v1/testtrigger"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+)
+
+func (s *Service) runWatcher(ctx context.Context) {
+	f := informers.NewSharedInformerFactory(s.clientset, 0)
+	podInformer := f.Core().V1().Pods()
+	deploymentInformer := f.Apps().V1().Deployments()
+	// daemonsetInformer := w.f.Apps().V1().DaemonSets()
+	// statefulsetInformer := w.f.Apps().V1().StatefulSets()
+	// serviceInformer := w.f.Core().V1().Services()
+	// ingressInformer := w.f.Networking().V1().Ingresses()
```

Comment:
do we enable them?

Reply:
maybe we can configure which one to listen?

Reply:
I enabled all of them